docs/architecture/grpc-for-wcf-developers/migrating-wcf-to-grpc.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Migrating from WCF to gRPC
+description: gRPC for WCF Developers | How to migrate different types of WCF service to gRPC
+author: markrendle
+ms.date: 09/02/2019
+---
+
+# Migrating from WCF to gRPC
+
+This chapter will look at how to convert different types of WCF service to the gRPC equivalent.
+
+- Simple Request-Reply operations to gRPC Unary RPC
+- One-way operations to gRPC Client Streaming RPC
+- Full Duplex services to gRPC Bi-directional Streaming RPC
```

Comment:
nit: I don't think you need to capitlaize "Unary", "Client Streaming", or "Bi-direction Streaming" here